components/Frame.js

```diff
@@ -1,7 +1,13 @@
 import React from "react";
 
-const Frame = () => {
-	return <div>Frame</div>;
+const Frame = ({ squareImage }) => {
+	return (
+		<img
+			className='ml-5 border-solid'
```

Comment:
what is the `ml-5` for?

Reply:
Oh, you added it before. I forgot to remove it. My bad.